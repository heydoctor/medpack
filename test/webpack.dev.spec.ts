import webpackBase from '../src/config/webpack.base';
import webpackDev from '../src/config/webpack.dev';

const paths = {};
const env = {};

const base = webpackBase({ paths, env });

test('webpack.dev', () => {
  expect(webpackDev({ base, paths, env })).toMatchSnapshot();
});
