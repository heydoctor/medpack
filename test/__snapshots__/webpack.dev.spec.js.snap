// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`webpack.dev 1`] = `
Object {
  "devtool": "cheap-module-source-map",
  "entry": Array [
    "/Users/kyle/dev/libraries/medpack/src/config/polyfills.js",
    "/Users/kyle/dev/libraries/medpack/node_modules/react-dev-utils/webpackHotDevClient.js",
    undefined,
  ],
  "mode": "development",
  "output": Object {
    "chunkFilename": "static/js/[name].chunk.js",
    "filename": "static/js/bundle.js",
    "pathinfo": true,
    "publicPath": "/",
  },
  "plugins": Array [
    HotModuleReplacementPlugin {
      "fullBuildTimeout": 200,
      "multiStep": undefined,
      "options": Object {},
      "requestTimeout": 10000,
    },
    CaseSensitivePathsPlugin {
      "fsOperations": 0,
      "options": Object {},
      "pathCache": Object {},
      "primed": false,
    },
  ],
}
`;
